database/sql/V057__add_fee_policy.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE PolicyKind AS ENUM ('priceimprovement', 'volume');
+
+CREATE TABLE fee_policies (
+  auction_id bigint NOT NULL,
+  order_uid bytea NOT NULL,
+  -- The order in which the fee policies are inserted and applied.
+  application_order SERIAL NOT NULL,
+
+  -- The type of the fee policy.
+  kind PolicyKind NOT NULL,
+  -- The fee should be taken as a percentage of the price improvement. The value is between 0 and 1.
+  price_improvement_factor double precision CHECK (price_improvement_factor >= 0 AND price_improvement_factor <= 1),
+  -- The fee should be taken as a percentage of the order volume. The value is between 0 and 1.
+  volume_factor double precision CHECK (volume_factor >= 0 AND volume_factor <= 1),
```

Comment:
What's the purpose of these checks? I think it we want to enforce invariants, we should do so in the rust code and not rely on database specific syntax (to me the fee policy database could simply be a key value store that allows looking up the fee policy for a specific order).

Reply:
Thought it was nice to have it since already available out of the box. Removed.